### ISSUE TITLE
Crashes the editor on compile 4.22.3

### ISSUE THREAD

**Issue by MMMarcis** [SkyInAPond/AdvancedBlueprintHotkeysWiki#1]
If I have opened the hotkey editor and later try to compile any blueprint the editor crashes. Sometimes it also crashes when spawning nodes.

`
UE4Editor_CoreUObject!UObjectBase::IsValidLowLevelFast() [d:\build\++ue4\sync\engine\source\runtime\coreuobject\private\uobject\uobjectbase.cpp:335]
UE4Editor_CoreUObject!FPersistentFrameCollectorArchive::operator<<() [d:\build\++ue4\sync\engine\source\runtime\coreuobject\private\uobject\uobjectglobals.cpp:3255]
UE4Editor_Core!FStructuredArchive::FSlot::operator<<() [d:\build\++ue4\sync\engine\source\runtime\core\private\serialization\structuredarchive.cpp:390]
UE4Editor_CoreUObject!UArrayProperty::SerializeItem() [d:\build\++ue4\sync\engine\source\runtime\coreuobject\private\uobject\propertyarray.cpp:218]
UE4Editor_CoreUObject!UStruct::SerializeBin() [d:\build\++ue4\sync\engine\source\runtime\coreuobject\private\uobject\class.cpp:870]
UE4Editor_CoreUObject!UStruct::SerializeBin() [d:\build\++ue4\sync\engine\source\runtime\coreuobject\private\uobject\class.cpp:854]
UE4Editor_Engine!UBlueprintGeneratedClass::AddReferencedObjectsInUbergraphFrame() [d:\build\++ue4\sync\engine\source\runtime\engine\private\blueprintgeneratedclass.cpp:1576]
UE4Editor_CoreUObject!FArchiveHasReferences::FArchiveHasReferences() [d:\build\++ue4\sync\engine\source\runtime\coreuobject\private\serialization\archivehasreferences.cpp:50]
UE4Editor_CoreUObject!FArchiveHasReferences::GetAllReferencers() [d:\build\++ue4\sync\engine\source\runtime\coreuobject\private\serialization\archivehasreferences.cpp:87]
UE4Editor_CoreUObject!FArchiveHasReferences::GetAllReferencers() [d:\build\++ue4\sync\engine\source\runtime\coreuobject\private\serialization\archivehasreferences.cpp:68]
UE4Editor_UnrealEd!FReplaceReferenceHelper::FindAndReplaceReferences() [d:\build\++ue4\sync\engine\source\editor\unrealed\private\kismet2\kismetreinstanceutilities.cpp:90]
UE4Editor_UnrealEd!FBlueprintCompileReinstancer::ReplaceInstancesOfClass_Inner() [d:\build\++ue4\sync\engine\source\editor\unrealed\private\kismet2\kismetreinstanceutilities.cpp:2115]
UE4Editor_UnrealEd!FBlueprintCompileReinstancer::BatchReplaceInstancesOfClass() [d:\build\++ue4\sync\engine\source\editor\unrealed\private\kismet2\kismetreinstanceutilities.cpp:1490]
UE4Editor_Kismet!FBlueprintCompilationManagerImpl::FlushReinstancingQueueImpl() [d:\build\++ue4\sync\engine\source\editor\kismet\private\blueprintcompilationmanager.cpp:1310]
UE4Editor_Kismet!FBlueprintCompilationManagerImpl::CompileSynchronouslyImpl() [d:\build\++ue4\sync\engine\source\editor\kismet\private\blueprintcompilationmanager.cpp:210]
UE4Editor_UnrealEd!FKismetEditorUtilities::CompileBlueprint() [d:\build\++ue4\sync\engine\source\editor\unrealed\private\kismet2\kismet2.cpp:766]
UE4Editor_Kismet!FBlueprintEditor::Compile() [d:\build\++ue4\sync\engine\source\editor\kismet\private\blueprinteditor.cpp:3258]
UE4Editor_Kismet!TBaseSPMethodDelegateInstance<0,FBlueprintEditor,0,void __cdecl(void)>::ExecuteIfSafe() [d:\build\++ue4\sync\engine\source\runtime\core\public\delegates\delegateinstancesimpl.h:355]
UE4Editor_Slate!FUICommandList::ExecuteAction() [d:\build\++ue4\sync\engine\source\runtime\slate\private\framework\commands\uicommandlist.cpp:99]
UE4Editor_Slate!SToolBarButtonBlock::OnClicked() [d:\build\++ue4\sync\engine\source\runtime\slate\private\framework\multibox\stoolbarbuttonblock.cpp:300]
UE4Editor_Slate!TBaseSPMethodDelegateInstance<0,SToolBarButtonBlock,0,FReply __cdecl(void)>::Execute() [d:\build\++ue4\sync\engine\source\runtime\core\public\delegates\delegateinstancesimpl.h:279]
UE4Editor_Slate!SButton::OnMouseButtonUp() [d:\build\++ue4\sync\engine\source\runtime\slate\private\widgets\input\sbutton.cpp:293]
UE4Editor_Slate!FEventRouter::Route<FReply,FEventRouter::FToLeafmostPolicy,FPointerEvent,<lambda_6c1f6b862bcfb82ea6dcab28fbcc9463> >() [d:\build\++ue4\sync\engine\source\runtime\slate\private\framework\application\slateapplication.cpp:271]
UE4Editor_Slate!FSlateApplication::RoutePointerUpEvent() [d:\build\++ue4\sync\engine\source\runtime\slate\private\framework\application\slateapplication.cpp:5598]
UE4Editor_Slate!FSlateApplication::ProcessMouseButtonUpEvent() [d:\build\++ue4\sync\engine\source\runtime\slate\private\framework\application\slateapplication.cpp:6203]
UE4Editor_Slate!FSlateApplication::OnMouseUp() [d:\build\++ue4\sync\engine\source\runtime\slate\private\framework\application\slateapplication.cpp:6176]
UE4Editor_ApplicationCore!FWindowsApplication::ProcessDeferredMessage() [d:\build\++ue4\sync\engine\source\runtime\applicationcore\private\windows\windowsapplication.cpp:1842]
UE4Editor_ApplicationCore!FWindowsApplication::DeferMessage() [d:\build\++ue4\sync\engine\source\runtime\applicationcore\private\windows\windowsapplication.cpp:2292]
UE4Editor_ApplicationCore!FWindowsApplication::ProcessMessage() [d:\build\++ue4\sync\engine\source\runtime\applicationcore\private\windows\windowsapplication.cpp:1522]
UE4Editor_ApplicationCore!FWindowsApplication::AppWndProc() [d:\build\++ue4\sync\engine\source\runtime\applicationcore\private\windows\windowsapplication.cpp:772]
user32
user32
UE4Editor_ApplicationCore!FWindowsPlatformApplicationMisc::PumpMessages() [d:\build\++ue4\sync\engine\source\runtime\applicationcore\private\windows\windowsplatformapplicationmisc.cpp:130]
UE4Editor!FEngineLoop::Tick() [d:\build\++ue4\sync\engine\source\runtime\launch\private\launchengineloop.cpp:3882]
UE4Editor!GuardedMain() [d:\build\++ue4\sync\engine\source\runtime\launch\private\launch.cpp:168]
UE4Editor!GuardedMainWrapper() [d:\build\++ue4\sync\engine\source\runtime\launch\private\windows\launchwindows.cpp:145]
UE4Editor!WinMain() [d:\build\++ue4\sync\engine\source\runtime\launch\private\windows\launchwindows.cpp:275]
UE4Editor!__scrt_common_main_seh() [d:\agent\_work\3\s\src\vctools\crt\vcstartup\src\startup\exe_common.inl:288]
kernel32
ntdll
`

**Comment by SkyInAPond** [comment]
Thank you for the report! Do you mean that it crashes primarily when you have the actual HotkeyEditor-window opened?

Thanks in advance // Skye

**Comment by MMMarcis** [comment]
It crashes if I have opened it at least once or is still open. Sometimes it needs to be opened and closed for the crash to happen on compile but it's fairly consistent. At first it wouldn't work at all because I didn't have Paper2D plugin installed. Does it rely on any other plugins I might have disabled? I also sen't you an email.

**Comment by SkyInAPond** [comment]
Ah alright! Thanks for the reproduction steps! I'll see too it that it'll be fixed asap! There shouldn't be any more plugin requirements, and I'll make sure Paper2d isn't a neccessity in the next release! :)

**Comment by MMMarcis** [comment]
Would it be possible to have an updated version before the Epic update? Those usually take longer and I'm still looking forward to using this.

**Comment by SkyInAPond** [comment]
Absolutely, I'll hopefully package a fixed version today to submit. I can send it to you via email at that point! // Skye

**Comment by MMMarcis** [comment]
That would be great!

**Comment by SkyInAPond** [comment]
I have sent you an email containing the newly packaged plugin. Please tell me if it works, and I'll submit it to Epic right away! :)

**Comment by MMMarcis** [comment]
I replied to your email.
